tests/loader_layer_tests.cpp

```diff
@@ -602,16 +1253,8 @@ TEST(OverrideMetaLayer, InvalidDisableEnvironment) {
                            "meta_test_layer.json");
 
     uint32_t layer_count = 0;
-    EXPECT_EQ(VK_SUCCESS, env.vulkan_functions.vkEnumerateInstanceLayerProperties(&layer_count, nullptr));
-    EXPECT_EQ(layer_count, 1U);
-
-    std::array<VkLayerProperties, 1> layer_props;
-    EXPECT_EQ(VK_SUCCESS, env.vulkan_functions.vkEnumerateInstanceLayerProperties(&layer_count, layer_props.data()));
-    EXPECT_EQ(layer_count, 1U);
-    EXPECT_TRUE(string_eq(layer_props[0].layerName, regular_layer_name));
-
-    InstWrapper inst{env.vulkan_functions};
-    inst.CheckCreate();
+    EXPECT_EQ(VK_ERROR_INITIALIZATION_FAILED, env.vulkan_functions.vkEnumerateInstanceLayerProperties(&layer_count, nullptr));
+    EXPECT_EQ(layer_count, 0U);
```

Comment:
This is fishy to me, since the explicit layer should be found normally, since its in the standard layer search paths.

Reply:
Ah, I caught a real bug.  I thought my code removed all layers appropriately but it was incorrect as you pointed out.  fixing.